src/MSAL.PCL/Internal/Cache/TokenCacheKey.cs

```diff
@@ -77,8 +71,6 @@ public override string ToString()
             stringBuilder.Append(MsalHelpers.Base64Encode(ClientId) + "$");
             // scope is treeSet to guarantee the order of the scopes when converting to string.
             stringBuilder.Append(MsalHelpers.Base64Encode(Scope.AsSingleString()) + "$");
```

Comment:
I think it is a good practice to protect the data. HomeOID could be anything for all we know..